rpi-eeprom-update

```diff
@@ -189,6 +189,30 @@ prepareImage()
    fi
 }
 
+runFlashrom()
+{
+   retries=5
```

Comment:
Mr pedantic in me would say that this should be named "attempts"

Reply:
Already then :)

Reply:
Just out of curiosity: if it fails on the 3rd attempt is it any more likely to work on the 4th or 5th attempt?

Reply:
We can't know that, but if the alternative is a unit that doesn't boot, how hard would you like us to try?

Reply:
Sorry, I was only being curious, I didn't mean to sound rude.